### PR TITLE
Added instrument, curve, government lookups

### PR DESCRIPTION
**Pull request by thmiceli**
This is a continuation of #64 but bases on a branch forked out of bloomberg/blpapi-http.

I took into account the feedback provided so far. The request map has been simplified since it was decided that we will keep the actual request name in the URL (see discussion for #59).
I also added some additional examples.

### CODE REVIEW

**Comment by thmiceli** [comment]
Just noticed that there is some overlap with what #73.

File: lib/blpapi-wrapper.ts [hunk 1]
```diff
@@ -279,8 +286,11 @@ export class Session extends events.EventEmitter {
         this.requests[correlatorId] = callback;
 
         this.openService(uri).then((): void => {
-            var responseEventName = reqName2RespName(name);
-            var requestName = name + 'Request';
+            if (!(name in REQUEST_NAMES_MAP)) {
```
Comment (ericlu88) [inline lib/blpapi-wrapper.ts:289]: @yupinghu is making similar changes here in #73. Maybe we can land that first and rebase against that?

Comment (ericvw) [inline lib/blpapi-wrapper.ts:289]: +1

**Comment by ericvw** [comment]
#73 landed.

**Comment by thmiceli** [comment]
Superseeded by #83

**Pull request closed without merging**